client/pages/sections/objectives/review.js

```diff
@@ -19,7 +19,7 @@ const ObjectivesReview = ({ playback, values, steps, goto, readonly, isFullAppli
         </Fragment>
       )
     }
-    <Playback playback={playback} />
+    <Playback playback={playback} isFullApplicationPdf={isFullApplication && pdf} />
```

Comment:
Is this potentially another place where we don't what kind of pdf it is and so should keep it simple and just pass the pdf prop?

I think yes.

Reply:
Also probably _don't_ actually pass it in at all if it can be loaded from the state in the child component.